tests/next_tests/integration_tests/feature_tests/iterator_tests/test_builtins.py

```diff
@@ -331,6 +333,9 @@ def test_can_deref(program_processor, stencil):
 @pytest.mark.parametrize("as_column", [False, True])
 def test_cast(program_processor, as_column, input_value, dtype, expected_value):
     program_processor, validate = program_processor
+    if program_processor == run_dace_iterator:
+        pytest.xfail("Not supported in DaCe backend: cast")
```

Comment:
This test will fail in c++ compilation, I don't know if validation should catch this problem earlier.